docs/components/filmstrip/README.md

```diff
@@ -17,36 +17,69 @@
 
 # Overview
     
-
 <cdr-doc-alert icon="info">Filmstrip is a Cedar cookbook component. Developers can choose from an optional range of subcomponents to create a custom version of it.</cdr-doc-alert>
 
 A filmstrip is a functional component container which allows users to preview multiple related pieces of content such as images, text, or cards in an allocated amount of space.
+  
+TODO code example should mimic this image (remove image once code sample is working as expected)
+<cdr-img src="https://i.imgur.com/Co3dIqI.jpg" alt="image of REI.com product recommendations filmstrip"/>
 
-## Customizing Filmstrip
 
-Because the Filmstrip is a cookbook of multiple subcomponents, its structure can be customized or overridden in a variety of ways. For example, not using a container, including a header, adding more or less content blocks in the preview, not including arrow pagination or using different types of content within the content blocks such as images, product tiles, animation or cards.
+## Anatomy of a Filmstrip
 
-### Use When
-- Providing a teaser of products, events, articles or categories to users
+Because the Filmstrip is a cookbook of multiple sub-components, its structure can be customized in a variety of ways.
```

Comment:
I moved this content under anatomy to reduce the amount of headings - there is some odd styling happening to h4s at the moment that I will track down